hydrography/add-enc-exchange-set/src/main/java/com/esri/samples/add_enc_exchange_set/AddEncExchangeSetSample.java

```diff
@@ -83,8 +88,8 @@ public void start(Stage stage) {
         }
       });
 
-      // create a map with the Oceans basemap and set it on the map view
-      ArcGISMap map = new ArcGISMap(Basemap.createOceans());
+      // create a map with the oceans basemap style and set it on the map view
+      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_OCEANS);
       mapView.setMap(map);
```

Comment:
If you decide to move the `stackPane` interaction I mentioned above, then we could move the creation of the map view down to here, so it looks the same as with the other samples`

Reply:
Thanks I've tidied this ordering up